app/src/main/java/org/p2p/wallet/common/ui/widget/actionbuttons/ActionButtonViewHolder.kt

```diff
@@ -0,0 +1,45 @@
+package org.p2p.wallet.common.ui.widget.actionbuttons
+
+import androidx.recyclerview.widget.RecyclerView
+import android.view.ViewGroup
+import org.p2p.wallet.R
+import org.p2p.wallet.databinding.ItemHomeButtonBinding
+import org.p2p.wallet.databinding.ItemSettingsInfoBinding
+import org.p2p.wallet.utils.viewbinding.inflateViewBinding
+
+class ActionButtonViewHolder(
+    private val binding: ItemHomeButtonBinding,
+    private val onButtonClicked: (ActionButton) -> Unit
+) : RecyclerView.ViewHolder(binding.root) {
+
+    constructor(parent: ViewGroup, onButtonClicked: (ActionButton) -> Unit) : this(
```

Comment:
org.p2p.wallet.history.ui.token.adapter.holders.TransactionViewHolder
for easier viewHolder